package writory.exception.authentication

import java.lang.RuntimeException

class UserFoundException : RuntimeException()
